iOS/DuckDuckGo/ProductionSubscriptionPurchaseDebugView.swift

```diff
@@ -0,0 +1,244 @@
+//
+//  ProductionSubscriptionPurchaseDebugView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Subscription
+import Core
+
+struct ProductionSubscriptionPurchaseDebugView: View {
+    @StateObject private var viewModel = ProductionSubscriptionPurchaseViewModel()
+    
+    var body: some View {
+        List {
+            warningSection
+            accountSection
+            usaSubscriptionsSection
+            rowSubscriptionsSection
+            statusSection
+        }
+        .navigationTitle("Buy Production Subs")
+        .onAppear {
+            Task {
+                await viewModel.loadExistingExternalID()
+            }
+        }
+        .alert("Purchase Result", isPresented: $viewModel.showAlert) {
+            Button("OK", role: .cancel) { }
+        } message: {
+            Text(viewModel.alertMessage)
+        }
+    }
+    
+    private var warningSection: some View {
+        Section {
+            Text("⚠️ WARNING ⚠️")
+                .font(.headline)
+                .foregroundColor(.red)
+            Text("This will attempt to purchase a REAL production subscription, bypassing all safety checks including existing subscription checks. Use with caution!")
+                .font(.subheadline)
+                .foregroundColor(.secondary)
```

Comment:
Tbt the warning is a bit over the top, as the only danger is for App Store downloaded app, on all other use cases the purchase will fail with error that no matching product with a given ID was found for the app.